server/server.js

```diff
@@ -8,8 +9,7 @@ const app = express();
 
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
-
-app.get('/', (req, res) => res.send('Hello World!'));
+app.use(express.static(path.join(__dirname, '../client/dist/')));
```

Comment:
app.use(express.static('client/dist'))

This work for me, do you have an special reason to use 'path'?